src/main/java/au/org/emii/ncdfgenerator/NcdfGenerator.java

```diff
@@ -0,0 +1,60 @@
+
+package au.org.emii.ncdfgenerator;
+
+import au.org.emii.ncdfgenerator.cql.ExprParser;
+import au.org.emii.ncdfgenerator.cql.IExprParser;
+import au.org.emii.ncdfgenerator.cql.IDialectTranslate;
+import au.org.emii.ncdfgenerator.cql.PGDialectTranslate;
+
+import java.sql.Connection;
+import java.io.InputStream;
+import java.io.FileInputStream;
+import java.io.OutputStream;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+
+public class NcdfGenerator
+{
+	final IExprParser parser;
+	final IDialectTranslate translate;
+	final ICreateWritable createWritable;
+	final String layerConfigDir;
+
+	public NcdfGenerator( String layerConfigDir, String tmpCreationDir  )
```

Comment:
`layerConfigDir`: @anguss00 and I were wondering about this one also yesterday.  When we were integration, we just hard coded a path *in the calling code* to this project - but obviously that's not right.  Where are these config files going to live - i.e. are they going to stay in this project?  Can the be packaged with this lib (in the jar)?  Again, the client would then not have to worry about configuring the lib.